src/main/java/it/pagopa/transactions/configurations/WebClientsConfig.java

```diff
@@ -44,6 +45,7 @@ public WebClient nodoWebClient(
             ObjectMapper mapper = new ObjectMapper();
             mapper.registerModule(new JavaTimeModule());
             mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+            mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
Remember to apply this fix also into event-dispatcher module for close payment retry requests